lib/metanorma/cli/site_generator.rb

```diff
@@ -11,6 +11,7 @@ def initialize(source, options = {})
         @asset_folder = options.fetch(:asset_folder, "documents").to_s
         @collection_name = options.fetch(:collection_name, "documents.xml")
         @manifest_file = find_realpath(options.fetch(:config, default_config))
+        @compile_options = options.select { |k, v| ["agree-to-terms", "no-install-fonts", "continue-without-fonts"].include?(k)}.map { |k, v| [k.to_sym, v] }.to_h
```

Comment:
Can we replace hyphens here with "_" in the symbol? i.e. `"--agree-to-terms"` => `:agree_to_terms`

Reply:
Good idea and thor support this out of the box! But because this will affect metanorma gem too I will report a separate issue for this https://github.com/metanorma/metanorma-cli/issues/218

Reply:
OK, thanks!